### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by ovari**
be able → enable
could not → unable to
history → historical
needed → required
not available → unavailable
webapp → the WebApp

- - -
This change is [Reviewable](https://reviewable.io/reviews/moneymanagerex/moneymanagerex/7045)

### CODE REVIEW

**Comment by n-stein** [comment]
The 3-to-1 ellipsis character affects so many strings, we might want to consider doing the ellipses conversion in our PO files as well then uploading the translations back to Crowdin so user's don't have to retranslate just for an ellipses character change.

**Comment by ovari** [comment]
Reverted the 3-to-1 ellipsis character conversion. Unsure how to:

> doing the ellipses conversion in our PO files as well then uploading the translations back to Crowdin so user's don't have to retranslate just for an ellipses character change.

**Comment by n-stein** [comment]
Just a procedural method to retain the existing translations with the character change. In fact, it would be better to do the ellipses change in a separate PR that only has those changes.

**Comment by ovari** [comment]
Can this patch please be merged and then another patch with ellipses can be done? Thank you

**Comment by georgeef** [comment]
You can also create a new PR depending on this PR. Create a new local branch based on this PR, commit the changes, push the new branch to your fork, create new PR. I did it a few times. For clarification, in the beginning of the new PR description I write something like: This PR depends on #...

File: src/maincurrencydialog.cpp [hunk 1]
```diff
@@ -432,8 +432,8 @@ void mmMainCurrencyDialog::OnListItemSelected(wxDataViewEvent& event)
                 else if (Model_CurrencyHistory::instance().find(Model_CurrencyHistory::CURRENCYID(m_currency_id)).size() > 0)
                 {
                     if (wxMessageBox(wxString::Format(_(
-                        "Historic rates for %1$s found, but \"Use currency history\" in options is disabled:\n"
-                        "click no and enable it or click yes to remove all historic rates for %2$s"),
+                        "Historical rates for %1$s found, but “Use historical currency” in options is disabled:\n"
```
Comment (n-stein) [inline src/maincurrencydialog.cpp:435]: I'm a bit surprised the removal of the escape character here didn't cause any issues.

Comment (ovari) [reply src/maincurrencydialog.cpp:435]: Does the smart quote require an escape character?

Comment (n-stein) [reply src/maincurrencydialog.cpp:435]: Ah, I didn't see the standard `"` character was replaced with the left/right quotation mark unicode symbols. Makes sense.

**Pull request merged**